src/rgw/librgw.cc

```diff
@@ -293,6 +293,25 @@ namespace rgw {
     int http_ret = s->err.http_ret;
 
     req->log_format(s, "http status=%d", http_ret);
+    switch (http_ret / 100) {
```

Comment:
@prashant-kr312 can't we use a dictionary that maps tag names to a tag handler to replace traditional case statements..